templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -1,262 +1,30 @@
 filesystem: "root"
 mode: 0644
-path: "/etc/crio/crio.conf"
+path: "/etc/crio/crio.conf.d/00-default"
 contents:
   inline: |
-    [crio]
```

Comment:
do you not still need this?

Reply:
yup, not needed anymore for drop in files